Framework/MDAlgorithms/test/Integrate3DEventsTest.h

```diff
@@ -527,7 +540,10 @@ class Integrate3DEventsTest : public CxxTest::TestSuite {
     for (double i = lower; i < upper; i += step) {
       for (double j = lower; j < upper; j += step) {
         for (double k = lower; k < upper; k += step) {
-          event_Qs.emplace_back(counts + d(gen), V3D(i, j, k));
+          double cts = counts + d(gen);
+          event_Qs.emplace_back(
+              std::make_pair(cts, cts),
+              V3D(i, j, k)); // is it right to add 1. error here?
```

Comment:
Could you help me understand this comment? It feels like a question to be answered and then remove from the code?